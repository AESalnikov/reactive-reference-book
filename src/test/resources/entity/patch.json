{
  "firstName": "Степан",
  "lastName": "Степанов",
  "middleName": "Степанович",
  "positionHeld": "Грузчик",
  "email": "error-ee@test.ru",
  "phoneNumber": "88000000004"
}