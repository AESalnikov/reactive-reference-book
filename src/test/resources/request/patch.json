{
  "email": "stepanov-ss@test.ru"
}