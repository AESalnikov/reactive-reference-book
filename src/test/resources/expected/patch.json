{
  "id": "00000000-0000-0000-0000-000000000004",
  "firstName": "Степан",
  "lastName": "Степанов",
  "middleName": "Степанович",
  "positionHeld": "Грузчик",
  "email": "stepanov-ss@test.ru",
  "phoneNumber": "88000000004"
}